packages/react/src/theming/extendTheming.ts

```diff
@@ -1,15 +1,18 @@
 import merge from 'deepmerge';
+import { AmplifyTheme, AmplifyThemeOverrides } from '@aws-amplify/ui';
 
-import { defaultTheme, defaultTokens } from './defaultTheming';
-import { DeepPartial, Theme, Tokens } from './types';
```

Comment:
Our own DeepPartial helper wasn't working properly for me. I added one from `type-fest` in the ui package and had that export a DeepPartial of the Theme.